README.md

```diff
@@ -1,4 +1,13 @@
-# Recommendations service repo.
+# Recommendations service.
 
+## Setup
 
-More to follow.
+- Clone the repo.
+- Start the Docker Daemon / Virtualbox. 
+- From the project root,
+    - if you're on an M1 mac, execute `vagrant up --provider docker`
+    - if you're on a Windows PC, execute `vagrant up --provider virtualbox`
+- After this, you will find a vm / docker container running on your system, inside which a docker container running postgres. 
+- To start the flask app, run `./scripts/start.sh` from the project directory. This will start the flask app on port `5000`.
```

Comment:
I see that the bash files make an assumption that the developer is working on local machine. This is fine for now. We shall change it later if required.

Reply:
Yeah, to startup in prod, we'll have to set it up differently.